pkg/game/board.go

```diff
@@ -222,3 +222,138 @@ func (board *board) checkCompleted(
 	}
 	return scoreReport, nil
 }
+
+/*
+It analyzes road directed by roadSide parameter.
+It doesn't analyze starting tile.
+returns: road_finished, score, [meeples on road], loop, sideFinishedOn
+param roadSide: always indicates only one cardinal direction!
+*/
+func (board *board) CheckRoadInDirection(roadSide side.Side, startTile elements.PlacedTile) (bool, int, []elements.Meeple, bool, side.Side) {
```

Comment:
What is "sideFinishedOn"? In ScoreRoadCompletion it's assigned to a variable called "loopSide", but it's not clear to me what it means

Reply:
Added more comments.
It is used for preventing checking the same road twice, as in test TestBoardScoreRoadLoopCrossroad, where the board looks like this:
  - 0 -
    1 - 2
    |	  |
  - 4 - 3
Fourth tile was placed last, and normally the road would be counted twice, but sideFinishedOn is passed to the function ScoreRoads to indicated that this is the same road, from another side.